### PR TITLE
Functionality for potential parameters identification

### PR DESCRIPTION
**Pull request by Ahmed-Aslam**
Currently, it is implemented to read parameters of EAM potentials in pyiron, can change any parameter and write it by default upon job run. 
**To be added : ** Quick additions "pair potentials, hybrid potential(any combination of pairwise and EAM)", long term "MEAM, Tersoff"

### CODE REVIEW

File: pyiron_atomistics/lammps/base.py [hunk 1]
```diff
@@ -169,13 +169,12 @@ def potential(self, potential_filename):
...
                                 ))
         self.input.potential.df = potential
+        self.input.potential.df
         if "Citations" in potential.columns.values:
             pot_pub_dict = {}
-            pub_lst = potential["Citations"].values[0]
-            if isinstance(pub_lst, str):
-                for p in ast.literal_eval(pub_lst):
-                    for k in p.keys():
-                        pot_pub_dict[k] = p[k]
```
Comment (jan-janssen) [inline pyiron_atomistics/lammps/base.py:178]: This change is required for https://github.com/pyiron/pyiron_atomistics/pull/22

File: pyiron_atomistics/lammps/base.py [hunk 2]
```diff
@@ -256,13 +256,16 @@ def get_potentials_for_structure(self):
         """
         return self.list_potentials()
 
-    @deprecate("use get_structure() instead")
```
Comment (jan-janssen) [inline pyiron_atomistics/lammps/base.py:259]: Why did you remove this?

**Review by jan-janssen — CHANGES_REQUESTED** [review]
Currently this pull request reverts a could of the recent changes which is not recommended.

File: pyiron_atomistics/lammps/base.py (hunk 2 above)
Comment (Ahmed-Aslam) [reply pyiron_atomistics/lammps/base.py:259]: I still have the previous version of base.py file, I'll update this in the next commit.
